elements/rh-pagination/rh-pagination.ts

```diff
@@ -24,6 +24,9 @@ const L2 = html`
 
 /**
  * Pagination
```

Comment:
should have a more verbose description here

Reply:
For sure. This will be addressed in https://github.com/RedHat-UX/red-hat-design-system/pull/993.